mlflow/spark.py

```diff
@@ -34,12 +36,67 @@ def log_model(spark_model, artifact_path, conda_env=None, jars=None):
     :param conda_env: Path to a Conda environment file. If provided, defines environment for the
     model. At minimum, it should specify python, pyspark, and mlflow with appropriate versions.
     :param jars: List of JARs needed by the model.
+    :param dfs_tmpdir: Temporary directory path on DFS or local FS if running in local mode. The
+    model will be writen in this destination and than copied to requested local path. This is
```

Comment:
"copied into the model's artifact directory" from this method's perspective

Reply:
good point, thx